books/free-programming-books-subjects.md

```diff
@@ -628,6 +629,11 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [ZeroMQ Guide](http://zguide.zeromq.org/page%3Aall)
 
 
+### object oriented programming
+
+* [Object Oriented Programming](https://www.cl.cam.ac.uk/teaching/0910/OOProg/OOP.pdf) - Dr Robert Harle (PDF)
```

Comment:
Is this resource free to be accessed by people who are not part of that university?

Reply:
@eshellman changes done

Reply:
check your spelling in the index, but please attend to my suggestions on the category to put this in.

Reply:
@eshellman  All done